packages/dds/tree/src/shared-tree-core/editManager.ts

```diff
@@ -57,6 +63,8 @@ export class EditManager<
 	TChangeset,
 	TChangeFamily extends ChangeFamily<TEditor, TChangeset>,
 > {
+	private readonly _events: ISubscribable<BranchTrimmingEvents> & IEmitter<BranchTrimmingEvents>;
```

Comment:
If you just do `private readonly _events = createEmitter<BranchTrimmingEvents>();` as an itializer here, you won't need these types and you also won't need to assign in the constructor.